hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -209,7 +211,7 @@
                             '''{eval_cmd} file=`echo {build_seed_file_path}`; \
                                if [ -f $file ]; then \
                                  while read line; do \
-                                   echo "--otp-seed $line"; \
+                                   echo "--otp-seed $line --lc-seed $line"; \
```

Comment:
Could OTP and LC seed be two different values?

Reply:
yes, although we only have one build seed at the moment. should we add something like an inline +1 expression here to make them different?

btwy, if we do that, it probs makes sense to make both the OTP and the LC seed different (e.g. +1 and +2) since the build seed is already used for generating the `RndCnst`.

Reply:
Right - I was thinking more on the lines of adding mask variables (that we can XOR with the build seed) in the Hjson that we can use to diversify the seed for different purposes. For that to work though, we will need to add support for `{eval_py}` to evaluate python code snippets within the HJson.

Reply:
Ok, something like that would be great to have. Let's land this first though (we can tweak later).

Reply:
Agreed.